script.js

```diff
@@ -1,4 +1,76 @@
+//global variables
+var gameMode = "PLAYER ONE ROLL MODE";
+var playerOneDiceArray = [];
+var playerTwoDiceArray = [];
+var finalValuePlayerOne;
+var finalValuePlayerTwo;
+
+var rollDice = function () {
+  return Math.floor(Math.random() * 6) + 1;
+};
+
+var diceRoll1 = rollDice();
+var diceRoll2 = rollDice();
+var diceRoll3 = rollDice();
+var diceRoll4 = rollDice();
```

Comment:
Interesting choice of rolling the dice on page load. 
However might be better to only rolling the dice when needed. Reason being our code has to be flexible for any changes (e.g. what if the player needs to roll 3 dices, what if there are more than 2 players)